tests/unit_tests/test_wave_03.py

```diff
@@ -131,7 +127,6 @@ def test_swap_items_from_their_empty_returns_false():
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
+    assert result == False
+    assert len(jolie.inventory) == 0
+    assert len(fatimah.inventory) == fatimah_len_l
```

Comment:
Great assertions, I would also suggest checking the inventory contents, since the length being correct doesn't guarantee that the individual elements are still what we expect.